Jenkinsfile

```diff
@@ -10,6 +12,12 @@ elifeLibrary {
         stage 'Build images', {
             checkout scm
             dockerComposeBuild(commit)
+            candidateVersion = dockerComposeRunAndCaptureOutput(
```

Comment:
This is now also starting the dependency (GROBID) and its output is captured too. I guess the best is to remove the dependency via `docker-compose.ci.yml`

Reply:
I don't think you can remove entities through the additional file, only add options

Reply:
Maybe GROBID should write to stderr rather than stdout if it makes a difference?

Reply:
I guess I could also override the command

Reply:
Actually it's not the output from GROBID itself but docker.

Reply:
moved the `depends_on` to `docker-compose.override.yml` instead